lib/runESLint.js

```diff
@@ -8,17 +8,18 @@ const runESLint = ({ fix = false, paths }) =>
 
     const cli = new EslintCLI({
       baseConfig: eslintConfig,
+      extensions: ['.ts', '.tsx', '.js', '.jsx'],
```

Comment:
Should we add `json` as per the path filter below?

Reply:
I did try that, but it's giving false positives on`json` files